.gitignore

```diff
@@ -47,3 +47,5 @@ package-lock.json
 # System Files
 .DS_Store
 Thumbs.db
+
+.iml
```

Comment:
Is this entry necessary being the `*.iml` pattern included in the file?

Reply:
Hmm 🤔 don't know, it's from IntelliJ. I can remove it.